# Define msg for camera height
int16 camera_height